deployments/local/ops/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  acme-grafana:
+    image: grafana/grafana:8.0.2
+    ports:
+      - 3000:3000
+    user: "104:1000"
+    volumes:
+      - ./run/grafana:/var/lib/grafana
+      - ../../../config/stage/ops/grafana/defaults.ini:/usr/share/grafana/conf/defaults.ini
```

Comment:
Is `ops` really a stage? I'd expected this to be part of `dev`. In concrete deployment scenarios I'd differentiate Keycloak environments that secure internal core services (JIRA, confluence, grafana etc.) from Keycloak environments that secure application domains.

We need to think more about how we can represent this in our example, for now the pseudo `ops` stage is fine, but I think we need an additional layer here to express the scope of the Keycloak environment, e.g. the domain (application / internal).

Reply:
Yes, i think this is a pretty nice point to discuss. I considered ops to be something central not necessarily aligned to the stages of domain apps or keycloak. Could be different and central and might be of minor interest to keycloak? Stages could be done via different labels? Lets discuss :)